src/view_page.cpp

```diff
@@ -1208,6 +1208,10 @@ void View::DrawSystemEditorialElement(DeviceContext *dc, EditorialElement *eleme
     // if (element->m_visibility == Visible) {
     //    DrawSystemChildren(dc, element, system);
     //}
+    if (element->Is(ANNOT)) {
+        Annot *annot = dynamic_cast<Annot *>(element);
```

Comment:
Had that at first, but removed it again as it just called `dc->AddDescription`. But it makes sense, especially when having `@plist` and wanting to do some nifty CSS pop-ups.